src/services/weight.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { BodyWeightType } from '../types';
+
+const BASEURL = 'https://wger-master.rge.uber.space/api/v2/weightentry/'
+
+interface ResponseType {
+    count: number,
+    next: number | null,
+    previous: number | null,
+    results: [BodyWeightType]
+}
+
+export const get_weights = async () => {
+    const {data: received_weights} = await axios.get<ResponseType>(BASEURL, {
+                headers: {
+                    Authorization: 'Token 31e2ea0322c07b9df583a9b6d1e794f7139e78d4'
```

Comment:
while this is ok for now, commiting secret keys is not a great idea :) Does react have some way of loading some kind of configuration? Then we could distinguish between development and production (where we can't hard code a specific token, but that is an issue for later)

Reply:
Oh my bad I forgot that.
Yeah using env variables should do it.